src/components/SingleArticle.jsx

```diff
@@ -0,0 +1,39 @@
+import { useEffect, useState } from "react"
+import { getArticleById } from "../../utils/utils"
+import { useParams } from "react-router-dom"
+import Home from "./Home"
+
+
+function SingleArticle(){
+    const [article, setArticle] = useState()
+    const {article_id} = useParams()
+
+    useEffect(()=>{
+        getArticleById(article_id).then((response)=>{
+            console.log(response.data.article)
+            setArticle(response.data.article)
+        })
+    },[article_id])
+
+    return (
+        <>   
+            <Home />
+            <div className="SingleArticle">
+            {article && <h2>{article.title}</h2> 
+            }
+            {article && <h3>Created: {article.created_at}</h3> 
```

Comment:
It would be better to have the timestamp formatted in a bit more of a human-readable way, than the default. Have a think about how you could manipulate the data yourself to do this or look into some packages that can do it for you 💡